auth_totp/models/res_users.py

```diff
@@ -13,6 +13,12 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
+    @classmethod
+    def _build_model(cls, pool, cr):
+        ModelCls = super(ResUsers, cls)._build_model(pool, cr)
+        ModelCls.SELF_WRITEABLE_FIELDS += ['mfa_enabled', 'authenticator_ids']
+        return ModelCls
+
```

Comment:
That would work too, but the new logic would now be triggered every time there's a write operation as opposed to once per registry, which doesn't seem like a good tradeoff.

FWIW, Odoo SA code has a couple examples of this being done in ``__init__``, but all the custom ``__init__`` logic has been [deprecated in v10](https://github.com/odoo/odoo/search?utf8=%E2%9C%93&q=SELF_WRITEABLE_FIELDS&type=) and ``_build_model`` is [now calling ``__init__`` just for legacy support](https://github.com/odoo/odoo/blob/10.0/odoo/models.py#L576), which is what let me to put it in ``_build_model``.